rust/operator-binary/src/druid_controller.rs

```diff
@@ -678,7 +676,13 @@ fn build_rolegroup_statefulset(
             &rolegroup_ref.role_group,
         ))
     });
-    pb.security_context(PodSecurityContextBuilder::new().fs_group(1000).build()); // Needed for secret-operator
+    pb.security_context(
+        PodSecurityContextBuilder::new()
+            .run_as_user(1000)
+            .run_as_group(1000)
+            .fs_group(1000) // Needed for secret-operator
```

Comment:
And other stuff... In other operators we don't mention this, so i would remove it in favor of consistency
```suggestion
            .fs_group(1000)
```